enhancements/README.md

```diff
@@ -0,0 +1,78 @@
+# Enhancements Tracking and Backlog
+
+Enhancement tracking repository for HyperShift.
+
+Inspired by the [OpenShift enhancement](https://github.com/openshift/enhancements) process.
+
+This directory provides a rally point to discuss, debate, and reach consensus
+for how HyperShift [enhancements](./enhancements) are introduced.
+
+Enhancements may take multiple releases to ultimately complete and thus provide
+the basis of a community roadmap.  Enhancements may be filed from anyone in the
+community, but require consensus from domain specific project maintainers in
+order to implement and accept into the release.
+
+For a quick-start, FAQ, and template references, see [the guidelines](guidelines/README.md).
+
+## Is My Thing an Enhancement?
+
+A rough heuristic for an enhancement is anything that:
+
+- impacts how a cluster is operated including addition or removal of significant
+  capabilities
+- impacts upgrade/downgrade
+- needs significant effort to complete
+- requires consensus/code across multiple domains/repositories
+- proposes adding a new user-facing component
+- has phases of maturity (Dev Preview, Tech Preview, GA)
+- demands formal documentation to utilize
+
+It is unlikely to require an enhancement if it:
+
+- fixes a bug
+- adds more testing
+- internally refactors a code or component only visible to that components
+  domain
+- minimal impact to distribution as a whole
+
+If you are not sure if the proposed work requires an enhancement, file an issue
+and ask!
+
+## When to Create a New Enhancement
+
+Create an enhancement here once you:
+
+- have circulated your idea to see if there is interest
+- (optionally) have done a prototype in your own fork
+- have identified people who agree to work on and maintain the enhancement
+  - many enhancements will take several releases to complete
+
+## Why are Enhancements Tracked
+
+As the project evolves, its important that the HyperShift community understands how we
+build, test, and document our work.  Individually it is hard to understand how
+all parts of the system interact, but as a community we can lean on each other
+to build the right design and approach before getting too deep into an
+implementation.
+
+## When to Comment on an Enhancement Issue
+
+Please comment on the enhancement issue to:
+- request a review or clarification on the process
+- update status of the enhancement effort
+- link to relevant issues in other repos
+
+Please do not comment on the enhancement issue to:
+- discuss a detail of the design, code or docs. Use a linked-to-issue or design PR
```

Comment:
This is interesting - is it standard process not to comment on design in an enhancement? There is a proposed section in the template called [Implementation Details/Notes/Constraints](https://github.com/openshift/hypershift/pull/511/files#diff-a4eb7b31979a31e26a883f2b3e7747a0f804969d5c780ed1002ec99141f6a595R105). If I have comments on the proposed design, am I supposed to create a new issue and point it to the enhancement PR and hope that it is merged after my issue is addressed?

Reply:
I was confused by this too, but upon further inspection, the blurb is referring to the enhancement _issue_ which is something different from the enhancement _PR_. The implication seems to be that the enhancement starts as an _issue_, leading to a _PR from the template_, and the _issue_ is apparently for tracking progress at the high level.

However, the readme doesn't describe the process of creating an issue, just a PR from the template. So maybe we should purge all references to the issue? Or if we want the issue -> PR process, maybe we should clarify?

Reply:
Ah - I see the distinction now. Requiring an issue to be created first will probably help non-contributors request a feature, but it also introduces additional overhead if its a hard requirement. I lean towards making projects more inclusive from the onset, but will let other chime in.

Reply:
I agree with you that the less overhead the better, we can always introduce an issue to the process later if we decide that would make any sense.

Reply:
I went ahead and removed this section entirely